streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/RackAwareTaskAssignor.java

```diff
@@ -139,7 +139,7 @@ public boolean populateTopicsToDescribe(final Set<String> topicsToDescribe,
         return populateTopicsToDescribe(fullMetadata, changelogPartitionsForTask, partitionsForTask, changelog, topicsToDescribe, racksForPartition);
     }
 
-    public static boolean populateTopicsToDescribe(final Cluster fullMetadata,
```

Comment:
ditto.

Reply:
I feel addressing this comment can fix all failed tests.

Reply:
Thanks, the solution is cleaner.